eng/dependabot/Packages.props

```diff
@@ -8,29 +26,26 @@
     the list of assemblies redistributed by MSBuild (non-MSBuild assemblies in the .vsix package).
      -->
   <ItemGroup>
-    <PackageReference Update="BenchmarkDotNet" Version="0.13.1" />
-    <PackageReference Update="LargeAddressAware" Version="1.0.5" />
-    <PackageReference Update="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="3.3.3" />
-    <PackageReference Update="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.36" />
-    <PackageReference Update="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="3.2.2146" PrivateAssets="All" />
-    <PackageReference Update="Microsoft.Win32.Registry" Version="5.0.0" />
-    <PackageReference Update="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Update="PdbGit" Version="3.0.41" />
-    <PackageReference Update="Shouldly" Version="3.0.0" />
-    <PackageReference Update="System.CodeDom" Version="6.0.0" />
-    <PackageReference Update="System.Private.Uri" Version="4.3.2" />
-    <PackageReference Update="System.Runtime" Version="4.3.1" />
-    <PackageReference Update="System.Security.Cryptography.Pkcs" Version="6.0.1" />
-    <PackageReference Update="System.Security.Cryptography.Xml" Version="6.0.0" />
-    <PackageReference Update="System.Security.Cryptography.X509Certificates" Version="4.3.2" />
+    <PackageReference Update="BenchmarkDotNet" Condition="'$(BenchmarkDotNetVersion)' == ''" Version="0.13.1" />
```

Comment:
From a maintenance perspective would it be better to have the two PackageReferences next to each other?

```
<PackageReference Update="BenchmarkDotNet" Version="0.13.1" />
<PackageReference Update="BenchmarkDotNet" Version="$(BenchmarkDotNetVersion)" Condition="'$(BenchmarkDotNetVersion)' != ''" />
```